tests/integration/targets/rabbitmq_vhost/tasks/tests_api.yml

```
@@ -0,0 +1,145 @@
+- block:
+  - set_fact:
+      vhost_name: /test
```

```suggestion
  - set_fact:
      vhost_name: /test
```
- I would suggest adding a few tasks with `check_mode: true` and subsequent checks that the state hasn't been changed. Is it possible?
- Also the checks just check what the module returns, but it can return something which is actually not true. Is it possible to check the actual state of the system after each test task? For example, with the shell module or with another module?